Xamarin.Forms.Platform.UAP/CellControl.cs

```diff
@@ -217,9 +217,10 @@ void SetDefaultSwitchColor()
 
 		void OnClick(object sender, PointerRoutedEventArgs e)
 		{
-			PointerPoint point = e.GetCurrentPoint(CellContent);
-			if (point.Properties.PointerUpdateKind != PointerUpdateKind.RightButtonReleased)
-				return;
+			//TODO : Implement PointerUpdateKind in Uno
```

Comment:
Can you open an issue in Uno so we keep track of this one ?

Reply:
I've got it. Raised over at https://github.com/unoplatform/uno/issues/1492 and doing impl now.